Fresh install cannot do 'make start'

I wanted to restart from scratch to make sure that I had all the most recent versions of everything after the last merge. but now after doing:

git pull (on master)
make clean
make env_clean
make conda_env
conda activate raven
make install
make start

make start is giving me this error:

```
(raven) ets@etsm:~/src/raven3/raven$ make start
Starting application ...
Traceback (most recent call last):
  File "/home/ets/anaconda3/envs/raven/bin/raven", line 33, in <module>
    sys.exit(load_entry_point('raven', 'console_scripts', 'raven')())
  File "/home/ets/anaconda3/envs/raven/bin/raven", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/home/ets/anaconda3/envs/raven/lib/python3.6/site-packages/importlib_metadata/__init__.py", line 96, in load
    module = import_module(match.group('module'))
  File "/home/ets/anaconda3/envs/raven/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/ets/src/raven3/raven/raven/__init__.py", line 10, in <module>
    from .wsgi import application  # noqa: F401
  File "/home/ets/src/raven3/raven/raven/wsgi.py", line 4, in <module>
    from .processes import processes
  File "/home/ets/src/raven3/raven/raven/processes/__init__.py", line 1, in <module>
    from .wps_raven import RavenProcess
  File "/home/ets/src/raven3/raven/raven/processes/wps_raven.py", line 7, in <module>
    from raven.utils import archive_sniffer, single_file_check
  File "/home/ets/src/raven3/raven/raven/utils.py", line 18, in <module>
    import fiona
  File "/home/ets/anaconda3/envs/raven/lib/python3.6/site-packages/fiona/__init__.py", line 84, in <module>
    from fiona.collection import BytesCollection, Collection
  File "/home/ets/anaconda3/envs/raven/lib/python3.6/site-packages/fiona/collection.py", line 9, in <module>
    from fiona.ogrext import Iterator, ItemsIterator, KeysIterator
ImportError: libtbb.so.2: cannot open shared object file: No such file or directory
Makefile:155: recipe for target 'start' failed
make: [start] Error 1 (ignorée)

```

Therefore I am unable to start the service to try and write notebooks, etc. Any ideas?

--- a/environment.yml
+++ b/environment.yml
@@ -43,6 +43,7 @@ dependencies:
 - pysheds
 - psycopg2
 - xskillscore
+- tbb
 - pip:
   - spotpy
   - urlpath

